### DISCUSSION TITLE
Mean Theoritical Schmid Factor for PolyCrystal IN718

### DISCUSSION THREAD

**Discussion by nargesmbg** [mtex-toolbox/mtex#1871] [Ask Anything]
hi all, 

I'm looking for a way to calculate kind of average schmid factor for my polycrystalline sample based on EBSD results I have and applied stress in (001)? 

how can I present one schmid factor for my sample? 

Best

**Comment by kilir** [comment]
Hi,
you could compute a Schmid factor for each pixel and average them or compute onw for each grain and average them, weighted by the grain area.
Cheers,
Rüdiger
